llvm/lib/CodeGen/MachineBlockPlacement.cpp

```diff
@@ -1714,8 +1714,9 @@ MachineBasicBlock *MachineBlockPlacement::selectBestCandidateBlock(
     return nullptr;
 
   bool IsEHPad = WorkList[0]->isEHPad();
+  bool ReverseOrder = IsEHPad;
```

Comment:
No need to define a new variable. The logic is extensively explained in the comments.

Reply:
I found this makes the code easier to understand as it makes things clear especially when not reading the long comment. But no strong opinion I can change this part back...